wolox-books/src/scss/forms.scss

```diff
@@ -0,0 +1,33 @@
+$form-width: 350px;
+$header-logo-width: 300px;
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: $form-width;
```

Comment:
Being the `form-width: 350px`, use `max-width` to set it to your variable, and then use `width: 100%` for it to only have it's maximum width available. If not, the layout might break on small screens 😄